packages/turf-invariant/index.js

```diff
@@ -4,32 +4,28 @@ import { isNumber } from '@turf/helpers';
  * Unwrap a coordinate from a Point Feature, Geometry or a single coordinate.
  *
  * @name getCoord
- * @param {Array<number>|Geometry<Point>|Feature<Point>} obj Object
+ * @param {Array<number>|Geometry<Point>|Feature<Point>} coord GeoJSON Point or an Array of numbers
  * @returns {Array<number>} coordinates
  * @example
  * var pt = turf.point([10, 10]);
  *
  * var coord = turf.getCoord(pt);
  * //= [10, 10]
  */
-export function getCoord(obj) {
-    if (!obj) throw new Error('obj is required');
-
-    var coordinates = getCoords(obj);
+export function getCoord(coord) {
+    if (!coord) throw new Error('coord is required');
+    if (coord.type === 'Feature' && coord.geometry !== null && coord.geometry.type === 'Point') return coord.geometry.coordinates;
+    if (coord.type === 'Point') return coord.coordinates;
+    if (coord.length >= 2 && coord[0].length === undefined && coord[1].length === undefined) return coord;
```

Comment:
@rowanwins & @stebogit This seems to be a 99% light & performant solution to test if the coordinates is correctly an Array of numbers.